### PR TITLE
APIS-4526 - Switch to principal/subordinate naming

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala [hunk 1]
```diff
@@ -56,36 +56,36 @@ class ProxyAwareApiDefinitionService @Inject()(localSvc: LocalApiDefinitionServi
...
-    val remoteFuture = remoteSvc.fetchExtendedDefinition(serviceName, thirdPartyDeveloperEmail)
+    val principalFuture = principal.fetchExtendedDefinition(serviceName, thirdPartyDeveloperEmail)
+    val subordinateFuture = subordinate.fetchExtendedDefinition(serviceName, thirdPartyDeveloperEmail)
 
     for {
-      maybeLocalDefinition <- localFuture
-      maybeRemoteDefinition <- remoteFuture
-      combined = combine(maybeLocalDefinition, maybeRemoteDefinition)
+      maybePrincipalDefinition <- principalFuture
+      maybeRemoteSubordinate <- subordinateFuture
```
Comment (adampridmore) [inline app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala]: Missed one!

File: app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala [hunk 2]
```diff
@@ -56,36 +56,36 @@ class ProxyAwareApiDefinitionService @Inject()(localSvc: LocalApiDefinitionServi
...
+      combined = combine(maybePrincipalDefinition, maybeRemoteSubordinate)
```
Comment (adampridmore) [inline app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala]: And another

File: app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala (hunk 1 above)
Comment (AndySpaven) [reply app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala]: Done

File: app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala (hunk 2 above)
Comment (AndySpaven) [reply app/uk/gov/hmrc/apidocumentation/services/ProxyAwareApiDefinitionService.scala]: Done

**Pull request merged**